lesson_19/homework_19_2.py

```diff
@@ -0,0 +1,46 @@
+import os.path
```

Comment:
Now code works only partially - Please organize your code as pytest class with tests. So there should be test_upload  , test_get, test_delete.